apps/b2b/package.json

```diff
@@ -8,24 +8,38 @@
     "start": "next start"
   },
   "dependencies": {
+    "@apollo/client": "^3.7.3",
+    "@hookform/resolvers": "^2.9.10",
+    "classnames": "^2.3.2",
+    "graphql": "15.8.0",
+    "graphql-ws": "^5.11.2",
+    "jsonwebtoken": "^9.0.0",
+    "lodash": "^4.17.21",
```

Comment:
don't include entire lodash. just use whatever lodash package we're actually using.

Reply:
where are we using lodash? i checked doesn't look like we're using?